PhysicsTools/NanoAOD/python/boostedTaus_cff.py

```diff
@@ -0,0 +1,205 @@
+import FWCore.ParameterSet.Config as cms
+from PhysicsTools.NanoAOD.common_cff import *
+from PhysicsTools.JetMCAlgos.TauGenJets_cfi import tauGenJets
+from PhysicsTools.JetMCAlgos.TauGenJetsDecayModeSelectorAllHadrons_cfi import tauGenJetsSelectorAllHadrons 
+
+##################### Updated tau collection with MVA-based tau-Ids rerun #######
+# Used only in some eras
+from PhysicsTools.NanoAOD.taus_updatedMVAIds_cff import *
+
+##################### User floats producers, selectors ##########################
+
+
+finalBoostedTaus = cms.EDFilter("PATTauRefSelector",
+    src = cms.InputTag("slimmedTausBoostedNewID"),
```

Comment:
In my test sequence this crashes because there are only slimmedTausBoosted. Is this maybe a remainder from the miniAOD fix? Please check.

Reply:
What era are you performing tests under? This sounds like the IDs are not being added and therefore the new slimmedTausBoostedNewID collection does not get made.

My command for reference:

cmsDriver.py step1 --filein "/store/relval/CMSSW_11_3_0_pre1/RelValTTbar_14TeV/MINIAODSIM/PU_113X_mcRun3_2021_realistic_v1-v1/10000/e41c73d6-dc6b-405a-8aa4-2f79d974a1ab.root" --mc --eventcontent NANOAODSIM --datatier NANOAODSIM --conditions auto:run2_mc --step NANO --nThreads 8 --era Run2_2016,run2_nanoAOD_106Xv2 -n 1000 --fileout file:testRelval.root

Reply:
I ran a 2016 sample but probably missed the run2_nanoAOD_106Xv2 modifier. Will try again. Thanks for the reference!

Reply:
Ok, with this modifier, it runs successfully. I think what is currently problematic is that the boosted taus are always added but the IDs only rerun with the given modifiers. But after rearranging the modifiers as stated earlier, all cases should be runnable (independently from the question whether boosted taus are produced in the respective cases)